packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.html

```diff
@@ -0,0 +1,45 @@
+<oui-modal
+    data-heading="{{:: 'pci_projects_project_storages_cold_archive_container_manage' | translate }}"
+    data-primary-action="$ctrl.onManageContainerClick()"
+    data-primary-disabled="$ctrl.isLoading"
+    data-secondary-action="$ctrl.cancel()"
+    data-on-dismiss="$ctrl.cancel()"
+    data-secondary-label="{{:: 'pci_projects_project_storages_cold_archive_containers_container_manage_close_label' | translate }}"
+    data-loading="$ctrl.isLoading"
+>
+    <p>
```

Comment:
**nitpick:** Isn't this redundant? `<p>`

Reply:
I used the block tag <p> in order to have a proper spacing for the inline tag <a>.